src/main/resources/crafter/studio/database/createDDL.sql

```diff
@@ -231,6 +231,26 @@ CREATE TABLE IF NOT EXISTS gitlog
   DEFAULT CHARSET = utf8
   ROW_FORMAT = DYNAMIC ;
 
+CREATE TABLE IF NOT EXISTS site_remote
+(
+  `id`                    BIGINT(20)    NOT NULL AUTO_INCREMENT,
+  `site_id`               VARCHAR(50)   NOT NULL,
+  `remote_name`           VARCHAR(255)   NOT NULL,
+  `remote_url`            VARCHAR(255)   NOT NULL,
+  `authentication_type`   VARCHAR(255)   NOT NULL,
```

Comment:
255? too long?

Reply:
Maybe. If your concern is space, I don't expect this table to grow too much